components/Ink/src/MDCInkView.m

```diff
@@ -49,6 +57,17 @@ - (void)commonMDCInkViewInit {
   self.backgroundColor = [UIColor clearColor];
   self.autoresizingMask = UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
   self.inkColor = self.defaultInkColor;
+  _usesLegacyInkRipple = YES;
+}
+
+- (void)layoutSubviews {
+  [super layoutSubviews];
+  CGRect inkBounds = CGRectMake(0, 0, CGRectGetWidth(self.frame), CGRectGetHeight(self.frame));
+
+  // When bounds change ensure all sublayers' bounds are changed too.
+  for (CALayer *layer in self.layer.sublayers) {
+    layer.bounds = inkBounds;
```

Comment:
Add a check to verify that the layer type is an ink layer.

Reply:
Done.